### DISCUSSION TITLE
`check_homogeneity` vs `check_heteroskedasticity`

### DISCUSSION THREAD

**Discussion by mattansb** [easystats/performance#409] [General]
I find it odd that it is possible, but I've often come across contradicting test results from homogeneity and heteroskedasticity tests in factorial designs:

``` r
library(performance)
#> Warning: package 'performance' was built under R version 4.1.3

m <- lm(mpg ~ interaction(vs, am),
        rbind(mtcars,mtcars[1:12,]))

check_homogeneity(m)
#> OK: There is not clear evidence for different variances across groups (Bartlett Test, p = 0.155).
check_heteroskedasticity(m)
#> Warning: Heteroscedasticity (non-constant error variance) detected (p = 0.028).
```

<sup>Created on 2022-03-27 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

I know these tests don't _exactly_ test the same thing, but...

Asking pedagogically - what would you do in such a case?

**Comment by DominiqueMakowski** [comment]
Interesting case, looking forward to read the answers

**Comment by bwiernik** [comment]
I never use the homogeneity test.
